src/types/formatted/qaGroup.ts

```diff
@@ -0,0 +1,16 @@
+import { PublishedParagraph } from './publishedEntity'
+import { FormattedParagraph } from '@/data/queries'
+
+type QaGroupQa = {
+  question: string
+  answers: FormattedParagraph[]
+  type: string
+  id: string
+}
+
+export type QaGroup = PublishedParagraph & {
+  questions: QaGroupQa[]
```

Comment:
I'm thinking more and more that we should get the QA node query/formatter set up properly and then type `questions` as `QuestionAnswer`, since that is how things are set up in Drupal. I know we want to get this over the line, so perhaps this is follow-on work, but I just don't think it makes a ton of sense to define `QaGroupQa` as a sort of quick-and-dirty knockoff of the actual node that we'll have to have eventually anyway.

Reply:
agreed. I think we can make that a fast follow so that we can get Vet Centers unblocked and in, but this should be cleaned up.

Reply:
Going to comment that QaGroupQa should be removed once QuestionAnswer is revisted.